tests/utils/exec/idps.go

```diff
@@ -0,0 +1,104 @@
+package exec
+
+import (
+	"context"
+	"fmt"
+
+	CON "github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/constants"
+	h "github.com/terraform-redhat/terraform-provider-rhcs/tests/utils/helper"
+)
+
+type IDPArgs struct {
+	ClusterID     string            `json:"cluster_id,omitempty"`
+	Name          string            `json:"name,omitempty"`
+	ID            string            `json:"id,omitempty"`
+	Token         string            `json:"token,omitempty"`
+	OCMENV        string            `json:"ocm_environment,omitempty"`
+	URL           string            `json:"url,omitempty"`
+	MappingMethod string            `json:"mapping_method,omitempty"`
+	GitHub        map[string]string `json:"github,omitempty"`
+	GitLab        map[string]string `json:"gitlab,omitempty"`
+	Google        map[string]string `json:"google,omitempty"`
+	Htpasswd      map[string]string `json:"htpasswd,omitempty"`
+	HtpasswdUsers []interface{}     `json:"htpasswd_users,omitempty"`
```

Comment:
I see this parameter covers also "Users"
htpasswdMap = []interface{}{map[string]string{"username": htpasswdUsername, "password": htpasswdPassword}}
and not actually used. Do you think it can be removed?

Reply:
removed 👍